cmd/dendritejs/main.go

```diff
@@ -69,10 +68,10 @@ func createFederationClient(cfg *config.Dendrite, node *go_http_js_libp2p.P2pLoc
 	return fed
 }
 
-func createP2PNode() (serverName string, node *go_http_js_libp2p.P2pLocalNode) {
-	hosted := "/dns4/ws-star.discovery.libp2p.io/tcp/443/wss/p2p-websocket-star"
-	_ = "/ip4/127.0.0.1/tcp/9090/ws/p2p-websocket-star/"
-	node = go_http_js_libp2p.NewP2pLocalNode("org.matrix.p2p.experiment", []string{hosted})
+func createP2PNode(privKey ed25519.PrivateKey) (serverName string, node *go_http_js_libp2p.P2pLocalNode) {
+	_ = "/dns4/ws-star.discovery.libp2p.io/tcp/443/wss/p2p-websocket-star"
```

Comment:
No-op

Reply:
atm I'm using this as a convenient way to swap between IPFS/local - though I won't ever use IPFS again because it barfs on ed25519 peer IDs, so will remove.

Reply:
Done